packages/protocol/truffle-config-parent.js

```diff
@@ -40,15 +36,23 @@ const defaultConfig = {
   from: OG_FROM,
   gas: gasLimit,
   gasPrice: 100000000000,
-  // maxFeePerGas: 975000000,
+  maxFeePerGas: 975000000000,
 }
 
-const freeGasConfig = { ...defaultConfig, ...{ gasPrice: 0 } }
+function readNmenomic(networkName) {
+  dotenv = require('dotenv').config({
+    path: require('path').resolve(__dirname, `../../.env.mnemonic.${networkName}`),
+  })
 
-// ipcProvider returns a function to create an IPC provider when called.
-// Use by adding `provider: ipcProvider(...)` to any of the configs below.
-function ipcProvider(path) {
-  return () => new Web3.providers.IpcProvider(path, net)
+  const privateKey = process.env.DEPLOYER_PRIVATKEY
```

Comment:
Could we update readme with this env variable ?

Reply:
Seems like a typo in the name of variable 😅 
Can it be named like `DEPLOYER_PK` or `DEPLOYER_PRIVATE_KEY` 🤔 ?